prerequisites.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+
+echo ""
+echo "Welcome! Before you can run our tool, you need to make sure you have all the packages and libraries installed !"
+
+echo ""
+#checks if caliper is installed under /bin, if not it will be installed and also runs the caliper bind to conigure ethereum globally
+
+cd cp_ws_1920/bin
+
+if hash "caliper" 2>/dev/null; then
+    echo 'Caliper is already installed'
+  else
+    echo 'Caliper is not installed in /bin'
+    echo 'We will process with installation'
+    npm install -g --only=prod @hyperledger/caliper-cli
+    echo 'We will run caliper bind to configure ethereum'
+   npx caliper bind --caliper-bind-sut ethereum --caliper-bind-sdk 1.2.1 --caliper-cwd ./ --caliper-bind-args="-g"
+  fi
+cd -
+
+echo ""
+echo " We will procees further"
+echo ""
+  if hash "jq" 2>/dev/null; then
+    echo 'jq is installed'
+  else
+    echo 'jq is missing.'
+    echo 'Please follow this link: https://stedolan.github.io/jq/download/  to install jq'
+  fi
+  
+  #Install and initialize the Cloud SDK using this link https://cloud.google.com/sdk/docs/?hl=de and install docker with this link https://docs.docker.com/install/linux/docker-ce/ubuntu/
+
+ if hash "gcloud" 2>/dev/null; then
+    echo 'Google cloud SDK is installed. Please wait a few seconds..'
+    gcloud auth configure-docker
```

Comment:
instead of doing this command, can we add the command to configure a gcp project? In the task is asked to configure the project the user will write in the config file.

Reply:
gcloud config set project my-project

Reply:
maybe i could live it up to the User to do that command "gcloud config set project my-project"
The project ID changes for everyone, unless the other task where the Proect ID is asked, can be submitted in this script here.